static/js/settings_users.js

```diff
@@ -601,6 +601,45 @@ function handle_human_form(tbody, status_field) {
         const modal = ret.modal;
         const fields_user_pills = ret.fields_user_pills;
 
+        const stream_ids = stream_data.get_default_stream_ids();
+        const default_streams = [];
+
+        for (const stream_id of stream_ids) {
+            const stream_name = stream_data.maybe_get_stream_name(stream_id);
+            default_streams.push({name: stream_name});
+        }
```

Comment:
Let's move this calculation inside the click-handler function; otherwise there's a race where this runs at load time, not at click time; the default streams could have been changed in the meantime.